tests/main/views/test_buyers.py

```diff
@@ -3846,8 +3846,8 @@ def test_404_if_user_is_not_brief_owner(self):
 
         assert res.status_code == 404
 
-    @pytest.mark.parametrize('status', ['withdrawn', 'draft', 'live', 'cancelled', 'unsuccessful', 'awarded'])
-    def test_404_if_brief_not_closed(self, status):
+    @pytest.mark.parametrize('status', ['withdrawn', 'draft', 'live'])
+    def test_404_if_brief_not_closed_and_award_flow_not_yet_completed(self, status):
```

Comment:
Doesn't _test_award_brief_get_returns_404_if_brief_not_closed_ already cover this cases or am I missing something?

Reply:
That's a separate view - there are 4 views in the award flow now:
- `award_or_cancel_brief` i.e. Have you awarded (Yes/No/Still Awarding)
- `award_brief` i.e. Who won the brief (choose from list of suppliers)
- `award_details` i.e. contract start date and value
- `cancel_brief` i.e. why did you cancel (cancelled/unsuccessful)

`test_404_if_brief_not_closed_and_award_flow_not_yet_completed` is for the first view, while `test_award_brief_get_returns_404_if_brief_not_closed` is for the second view. The naming is not the most consistent thing ever 😿